src/routes.ts

```diff
@@ -167,73 +223,80 @@ export const createRouter = (ctx: AppContext) => {
 
       // Map user DIDs to their domain-name handles
       const didHandleMap = await ctx.resolver.resolveDidsToHandles(
-        statuses.map((s) => s.authorDid)
+        statuses.map((s) => s.authorDid),
       )
 
       if (!agent) {
         // Serve the logged-out view
-        return res.type('html').send(page(home({ statuses, didHandleMap })))
+        res.type('html').send(page(home({ statuses, didHandleMap })))
+        return
```

Comment:
I believe all of these are alright from a typing perspective.  In my view combining these when possible is going to be nicer to read, and they will remain in the style of the published guide.
```suggestion
        return res.type('html').send(page(home({ statuses, didHandleMap })))
```